docs/conf.py

```diff
@@ -51,10 +51,11 @@
 # other places throughout the built documents.
 #
 # The short X.Y version.
-version = '%s.%s' % tuple(rqmt.version.split('.')[:2])
+version = '%s.%s' % tuple(rqmt.split('.')[:2])
 # The full version, including alpha/beta/rc tags.
 release = rqmt.version
```

Comment:
This will break with `AttributeError: 'str' object has no attribute 'version'` because you already got `.version` on line 21, which returns a `str`

Reply:
Ouch, right, thanks for noticing it!